addons/mail/static/src/new/attachments/attachment_uploader_hook.js

```diff
@@ -108,6 +108,14 @@ export function useAttachmentUploader(pThread, message, isPending = false) {
         }
         uploadingAttachmentIds.delete(tmpId);
         abortByAttachmentId.delete(tmpId);
+        if (upload.xhr.status === 413) {
+            notification.add("File too large", { type: "danger" });
```

Comment:
text should be translated, also it would help user to specify the current vs max size

Reply:
> also it would help user to specify the current vs max size

I don't think we have the server max size stored somewhere? I think it's handled by Nginx?

Reply:
We have `max_file_upload_size`, it's not guaranteed that nginx is configured the same, but it's the best we have. If they don't match it's a missconfig from the server admin.